student/views.py

```diff
@@ -1,3 +1,23 @@
-# from django.shortcuts import render
+from django.shortcuts import render, HttpResponseRedirect
+from .models import Student
+from .forms import UpdateStudentAccountSettingsForm
 
-# Create your views here.
+
+def account_update_success(request):
+    is_student = Student.is_student(request.user.id)
+    return render(request, 'account_settings_update_success.html', {"is_student": is_student})
+
+
+def update_student_account_settings_view(request):
+    context = {}
+    try:
+        obj = Student.objects.get(user_id=request.user.id)
```

Comment:
meaningful variable names? i. e `student_object`

Reply:
I'll change it